client/src/client.rs

```diff
@@ -351,6 +351,22 @@ pub trait RpcApi: Sized {
         self.call("getmininginfo", &[])
     }
 
+    fn get_block_template(
+        &self,
+        mode: json::GetBlockTemplateOptionsModes,
```

Comment:
I'd suggest dropping the word `Options` from these names to make them slightly smaller.